brevent/src/main/res/values-es/strings.xml

```diff
@@ -2,35 +2,33 @@
 <resources xmlns:tools="http://schemas.android.com/tools">
 
     <string name="app_name">Brevent</string>
-    <string name="app_name_debug">Brevent Comunidad</string>
+    <string name="app_name_debug">Comunidad Brevent</string>
```

Comment:
Don't change the order. Please make Brevent first, this is for self-building Brevent.